============================= test session starts ==============================
platform darwin -- Python 3.12.4, pytest-8.1.1, pluggy-1.4.0
rootdir: /Users/claudiobrandolino/llamda/llamda-fn
configfile: pyproject.toml
plugins: asyncio-0.23.8, anyio-4.3.0
asyncio: mode=Mode.STRICT
collected 26 items / 1 error

==================================== ERRORS ====================================
____________________ ERROR collecting tests/test_llamda.py _____________________
../../Library/Caches/pypoetry/virtualenvs/llamda-fn-T-M8I_mI-py3.12/lib/python3.12/site-packages/_pytest/python.py:520: in importtestmodule
    mod = import_path(
../../Library/Caches/pypoetry/virtualenvs/llamda-fn-T-M8I_mI-py3.12/lib/python3.12/site-packages/_pytest/pathlib.py:584: in import_path
    importlib.import_module(module_name)
/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1387: in _gcd_import
    ???
<frozen importlib._bootstrap>:1360: in _find_and_load
    ???
<frozen importlib._bootstrap>:1331: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:935: in _load_unlocked
    ???
../../Library/Caches/pypoetry/virtualenvs/llamda-fn-T-M8I_mI-py3.12/lib/python3.12/site-packages/_pytest/assertion/rewrite.py:169: in exec_module
    source_stat, co = _rewrite_test(fn, self.config)
../../Library/Caches/pypoetry/virtualenvs/llamda-fn-T-M8I_mI-py3.12/lib/python3.12/site-packages/_pytest/assertion/rewrite.py:351: in _rewrite_test
    tree = ast.parse(source, filename=strfn)
/opt/homebrew/Cellar/python@3.12/3.12.4/Frameworks/Python.framework/Versions/3.12/lib/python3.12/ast.py:52: in parse
    return compile(source, filename, mode, flags,
E     File "/Users/claudiobrandolino/llamda/llamda-fn/tests/test_llamda.py", line 177
E       result = llamda_instance.("Just give me a simple response.")
E                                ^
E   SyntaxError: invalid syntax
=========================== short test summary info ============================
ERROR tests/test_llamda.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.09s ===============================
